### PR TITLE
feat: created home page

### PR DESCRIPTION
**Pull request by nakulsharma15**
Feat: Created Home Page containing the following features:
  
1. Create a new note
2. Edit a note
3. Pin a note
4. Unpin a note

### CODE REVIEW

**Review by vaibhav-mahalle — APPROVED** [review]
Liked the clean code. Liked how you break the UI into components, everything looks good to me no changes required.

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+@import url("https://altur-ui.netlify.app/stylesheets/main.css");
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+@import url("https://pro.fontawesome.com/releases/v5.10.0/css/all.css");
```
Comment (rahulcs754) [inline src/App.css:3]: you can put these link to public/index.html that is good

File: src/Components/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Header from "./Header";
+import Footer from "./Footer";
+import Aside from "./Aside";
+import NoteCard from "./NoteCard";
+import NoteEditor from "./NoteEditor";
+import PinnedNoteCard from "./PinnedNoteCard";
```
Comment (rahulcs754) [inline src/Components/index.js:6]: you can export just replace import to export
no need to 8 line

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean small suggestion

**Pull request merged**